data.sql

```diff
@@ -8,3 +8,20 @@ VALUES
   ('Pikachu', '2021-01-07', 1, false, 15.04),
   ('Devimon', '2017-05-12', 5, true, 11);
 
+INSERT INTO animals 
+  (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Charmander', '2020-08-02', 0, false, 11),
+  ('Plantmon', '2021-11-15', 2, true, 5.7),
+  ('Squirtle', '1993-04-02', 3, false, 12.13),
+  ('Angemon', '2005-06-12', 1, true, 45),
+  ('Boarmon', '2005-06-07', 7, true, 20.4),
+  ('Blossom', '1998-08-13', 3, true, 17),
+  ('Ditto', '2022-05-14', 4, true, 22);
+
+-- CORRECT DATA
+BEGIN;
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE name IN ('Charmander', 'Plantmon', 'Squirtle', 'Angemon');
+COMMIT;
```

Comment:
- [ ] As required by the previous reviewer, you suppose to correct the entered data values, not make a transaction to correct them, Please fix that.